tests/drivers/build_all/sensor/sensors_trigger_own.conf

```diff
@@ -45,3 +45,4 @@ CONFIG_TMP007_TRIGGER_OWN_THREAD=y
 CONFIG_VCNL4040_TRIGGER_OWN_THREAD=y
 CONFIG_WSEN_HIDS_TRIGGER_OWN_THREAD=y
 CONFIG_WSEN_TIDS_TRIGGER_OWN_THREAD=y
+CONFIG_TCN75A_TRIGGER_OWN_THREAD=y
```

Comment:
Need to do `sensors_trigger_global.conf` and `sensors_trigger_none.conf` too

Reply:
Can you move this line up to preserve alpha order? Same for the other trigger confs